### DISCUSSION TITLE
Neo Geo CD support

### DISCUSSION THREAD

**Discussion by sreuter** [city41/neosdconv#10] [Ideas]
Hi @city41! I'm not sure if you considered this already in the past, but it would be awesome to get support for creating Neo Geo CD .neo files at some point, as mentioned here: https://wiki.terraonion.com/index.php/NeoSD_Pro_CD_Setup_Guide

I'm currently looking into adding CD support to our game, so this would me a massive help :)

**Comment by city41** [comment]
That would be awesome. But as far as I know, Terraonion and Razoola haven't released any info on how CD .neos are made. I doubt I'd be able to reverse engineer the process. I also think since they've only released 50 games so far, that might suggest the process varies a bit from game to game.

**Comment by city41** [comment]
Converted into a discussion. Didn't realize you needed to enable discussions per repo.

**Comment by city41** [comment]
For anyone interested, I am attempting to reverse engineer the cd format here

https://github.com/city41/neosdcd
